project_1_1.py

```diff
@@ -0,0 +1,47 @@
+import time
+
+name = input ("Please Enter Your Name ")
+print ("Hi "+ name +" Please Choose 2 Number ")
+str_number_1 = input("First Number ")
+str_number_2 = input("Second Number ")
+print("Than You For Choosing Two Numbers The First "+ str_number_1 +" The Second is " +str_number_2)
+int_number_1 = int (str_number_1)
+int_number_2 = int (str_number_2)
+if (int_number_1%2==0):
+    print("The First Number is Even ")
+else:
+    print("The First Number is Odd ")
```

Comment:
1. שים לב שהמשפט "The First Number is " מופיע גם בשורה 11 וגם בשורה 13. זה שכפול קוד, זה לא טוב. צריך לתקן את זה. תראה בבקשה את השיעור על שכפולי קוד כדי לראות איך מסדרים את זה
2. יש רווח בסוף השורות פרינט - לא צריך